nntrainer/tensor/blas_interface.cpp

```diff
@@ -182,7 +182,7 @@ static void scopy_float16_to_float32(const unsigned int N, const _FP16 *X,
   unsigned int incy = abs(incY);
   unsigned int incx = abs(incX);
 
-#if (defined USE__FP16 && USE_NEON)
+#if (defined(USE__FP16) && USE_NEON)
```

Comment:
Because you've added ```-DUSE_NEON=1``` in meson.build, it should work, but in case you didn't do so in Android build, (I don't see -DUSE_NEON in jni directory) please correct this, too.

Reply:
Apparently I missed it..
Fixed.